README.txt

```diff
@@ -32,9 +32,70 @@ Portability
 
 Ease of use
     The code follows the `Python Packaging User Guide
-    <http://python-packaging-user-guide.readthedocs.org/en/latest/>`_. This will
-    make installing the ``ga4gh`` reference code very easy across a range of
-    operating systems.
+    <http://python-packaging-user-guide.readthedocs.org/en/latest/>`_. 
+    Specifically, pip is used to handle python package dependencies (see below 
+    for details). This allows for easy installation of the ``ga4gh`` reference code 
+    across a range of operating systems.
+
+
+*******************************
+Installing the reference server
+*******************************
+
+Prerequisites:
+
+- Python 2.7 or higher,
+- Berkeley DB together with include and lib files (version 48 or higher),
+- Virtualenv (or another python sandboxing tool) is highly recommended.
+
+General installation procedure:
+
+- Install Berkeley DB (version 48 or higher) using your system's preferred 
+  package manager, see the `wormtable help page 
+  <https://pypi.python.org/pypi/wormtable>`_ for platform-specific details.
+  In one developer's experience, installing db48 was preferable to db53 
+  on MacOS X via MacPorts. For example (on MacOS X with MacPorts):
+
```

Comment:
I realise a lot of people use macs, but these are very mac specific instructions. For Debian/Ubuntu we can just `sudo apt-get install libdb-dev` and we're done. Can't we just provide a link to the wormtable documentation here?